packages/nextjs/components/cursor/FlyingReaction.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from "./FlyingReaction.module.css";
+
+type Props = {
+  x: number;
+  y: number;
```

Comment:
seems like there's quite a few types with `x: number; y: number` here. can we reuse (export) or refactor (have a `Point` interface/type that these other types use) somehow?

Reply:
Would it be similar to this interface in canvas.tsx
Should we have a separate file for types? 
```
interface PixelXY {
  x: number;
  y: number;
}
```

Reply:
in `liveblocks.config.ts`
The types `RoomEvent` and `Presence` must be JsonObjects so they can't have nested types 
<img width="824" alt="image" src="https://github.com/jaxonL/copix/assets/60107091/8793dada-b7b4-4a2a-be8d-5cbbcf682182">